index.html

```diff
@@ -0,0 +1,214 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="css/styles.css">
+  <link rel="shortcut icon" href="icons/Game_Developer_favicon.png" type="image/x-icon">
+  <script src="https://kit.fontawesome.com/30b7066c1c.js" crossorigin="anonymous"></script>
+  <link href="https://fonts.googleapis.com/css2?family=Lato:wght@100;300;400;700;900&display=swap" rel="stylesheet">
+  <title>Games Dev</title>
+</head>
+<body>
+
+  <!--* HEADER *-->
+
+  <header>
+    <nav class="first-nav">
+      <ul class="brands-nav">
+        <li>
+          <a href="#">
+            <i class="fa-brands fa-facebook"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <i class="fa-brands fa-twitter"></i>
+          </a>
+        </li>
+        <li>
+          <a href="#">
+            <i class="fa-brands fa-youtube"></i>
+          </a>
+        </li>
+      </ul>
+      <ul class="nav-links">
+        <li>
+          <a href="#">English</a>
+        </li>
+        <li>
+          <a href="#">My Page</a>
+        </li>
+        <li>
+          <a href="#">Logout</a>
+        </li>
+      </ul>
+    </nav>
+    <nav class="second-nav">
+      <ul class="second-nav-ul-icon">
+        <li>
+          <a href="index.html"><img src="icons/main-icon.png" alt="Game Dev Icon"></a>
+        </li>
+      </ul>
+      <ul class="second-nav-ul-links">
+        <li>
+          <a href="about.html">About</a>
+        </li>
+        <li>
+          <a href="#">Program</a>
+        </li>
+        <li>
+          <a href="#">Join</a>
+        </li>
+        <li>
+          <a href="#">Sponsor</a>
+        </li>
+        <li>
+          <a href="#">News</a>
+        </li>
+        <li>
+          <a href="#">GD Campaign</a>
+        </li>
+      </ul>
```

Comment:
- [ ] Good job by implementing the menu, I think it can be a good idea if you add also the Home page link to your menu so the user can access the Home page when they are on the About page without clicking on the logo 😊 😉